docs-site/src/pages/docs/getting-started.js

```diff
@@ -1,13 +1,67 @@
 import * as React from 'react'
+import { Section } from '../../components/Section'
 import { PageHeader, Subheader } from '../../components/Headers'
+import { CodeBlock, CodeFile } from '../../components/CodeBlock'
 
 export default () => {
   return (
     <div>
       <PageHeader>Getting Started</PageHeader>
-      <p>
-        TODO
-      </p>
+      <Section title="Installation">
+        <ol>
+          <li>
+            <p>
+              Initialize a new npm project.
+            </p>
+            <CodeBlock language="bash">
+              $ npm init
+            </CodeBlock>
+          </li>
+          <li>
+            <p>
+              Install bright-js-framework and React.
+            </p>
+            <CodeBlock language="bash">
+              $ npm install react @brightinteractive/bright-js-framework
```

Comment:
Do we require a specific version of react? Are we using 16?

Reply:
Using 16. Thinking it might actually make more sense to bundle React with the framework, though...

Reply:
In practice, the user needs to use the same React major version as the framework, so there's no reason they'd really want to bring their own version.

Reply:
@chrisdevereux For major (and even minor version) this is definitely true. But what about if there's a PATCH release that the user needs (bugfix/security/etc) - do we want to force them to wait for us to include the update?

Reply:
Because of the way that npm versioning works, this won't be a problem. We declare React >=16 <17 as a dependency of this package, the latest gets pulled into user projects on install, then whenever the user upgrades their dependencies, they get the latest version that satisifies the version constraint.